docs/getting-started.md

```diff
@@ -105,3 +110,35 @@ Or to start the auto-restarting development server:
 ```sh
 make start-dev
 ```
+
+Run Docker for development
+----------------------
+
+The `Dockerfile.dev` and `docker-compose.yml` file at the root of the project work together to start the
+development server with a MySQL database.
+
+Note: so far this has been built with a bias for ARM64/Apple Silicon machines. 
+
+Prerequisites:
+
+- [Docker Desktop](https://docs.docker.com/desktop/) or CLI
+
+### Setup
+1. Copy the `.env.defaults` file to `.env` and edit the variables to your liking. 
+1. Ensure the variables match the values in the `docker-compose.yml` file.
+1. Create a RSA JWT private key at the root of the project with `openssl genrsa -out ./jwt_secret.key 4096`
+It will be read by the server from as the `JWT_PRIVATE_KEY` environment variable.
+
+### Commands
+
+`docker login` to login into the Github container registry. See ["Authenticating to the Container Registry"](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-to-the-container-registry)
```

Comment:
Could be wrong but I don't think this is needed

Reply:
ah I added this only cause I had to

Reply:
It's weird because i thought docker compose (in this case) uses the local dockerfile and builds it. Why would it even need access to github at all?

Reply:
There's a likelihood I'm missing a `$HOME/.docker/config.json` on my laptop despite being almost always logged into github with my terminal. 

I think this login thing came from the original Dockerfile having a gcr.io image and probably needed to be logged in to pull it